sql/src/test/java/io/crate/integrationtests/LuceneQueryBuilderIntegrationTest.java

```diff
@@ -313,7 +314,10 @@ public void testWhereNotEqualAnyWithLargeArray() throws Exception {
 
     @Test
     public void testNullOperators() throws Exception {
-        DataType<?> type = randomType();
+        DataType<?> type;
+        do {
+           type = randomType();
+        } while (type == DataTypes.INTERVAL);
```

Comment:
Why is this necessary?

Reply:
Because you cannot have a column of type interval.

```
cr> create table t1(c interval);
SQLActionException[SQLParseException: Cannot use the type `interval` for column: c]

SQLActionException: 400 Bad Request 4000 SQLParseException: Cannot use the type `interval` for column: c
	at io.crate.analyze.AnalyzedColumnDefinition.validate(AnalyzedColumnDefinition.java:369)
	at io.crate.analyze.AnalyzedTableElements.finalizeAndValidate(AnalyzedTableElements.java:319)
	at io.crate.planner.node.ddl.CreateTablePlan.bind(CreateTablePlan.java:150)
	at io.crate.planner.node.ddl.CreateTablePlan.executeOrFail(CreateTablePlan.java:97)
	at io.crate.planner.Plan.execute(Plan.java:74)
       ...
```